src/xrpld/app/tx/detail/InvariantCheck.cpp

```diff
@@ -380,7 +380,8 @@ AccountRootsDeletedClean::finalize(
     // transaction processing results, however unlikely, only fail if the
     // feature is enabled. Enabled, or not, though, a fatal-level message will
     // be logged
-    bool const enforce = view.rules().enabled(featureInvariantsV1_1);
+    [[maybe_unused]] bool const enforce =
```

Comment:
Would this be a good candidate for `SOMETIMES`? If the fuzzer never can get the enforce variable to be true, perhaps it indeed is unused.

Reply:
The `[[maybe_unused]]` attribute means that the compiler should not warn when the variable is never used. This variable `enforce` is only ever used inside an `ASSERT` (down below in this function), meaning the compiler will see it being used in any Debug build (including `voidstar` one) but will _not_ see it being used in a Release build. That's how we use `[[maybe_unused]]` in several other location in this project. On the other hand `SOMETIMES` disables check altogether, and that's not what we want - since before this PR we had actual `assert` in this location.